internal/provider/resource_target.go

```diff
@@ -73,8 +74,22 @@ func resourceTarget() *schema.Resource {
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
-			targetApplicationCredentialSourceIdsKey: {
-				Description: "A list of application credential source ID's.",
+			"application_credential_source_ids": {
+				Description:   "A list of application credential source ID's.",
+				Type:          schema.TypeSet,
+				Optional:      true,
+				Deprecated:    "Please use 'brokered_credential_source_ids' instead",
+				ConflictsWith: []string{targetBrokeredCredentialSourceIdsKey},
```

Comment:
Does this conflict not apply the other way too?

Reply:
It absolutely does